### PR TITLE
Create E0768

### PR DESCRIPTION
**Pull request by GuillaumeGomez**
r? @Dylan-DPC

### CODE REVIEW

File: src/librustc_error_codes/error_codes/E0767.md [hunk 1]
```diff
@@ -0,0 +1,13 @@
+A number in non-decimal base has no digits.
```
Comment (teymour-aldridge) [inline src/librustc_error_codes/error_codes/E0767.md:1]:
```suggestion
A number in a non-decimal base has no digits.
```

**Comment by GuillaumeGomez** [comment]
@teymour-aldridge Thanks, updated!

**Comment by teymour-aldridge** [comment]
No problem!

**Pull request merged**